src/unitStats/mappingAbilities.ts

```diff
@@ -37,10 +37,15 @@ type AbilitiesUiData = {
   iconName: string; // Could be empty.
   symbolIconName: string; // Could be empty.
   /* Locstring fields. */
-  helpText: string | null;
-  briefText: string | null;
-  screenName: string | null;
-  extraText: string | null; // Could be empty (Set as $0).
+  helpText: string;
+  briefText: string;
+  screenName: string;
+  extraText: string; // Could be empty (Set as $0).
```

Comment:
This is one thing I am not sure about.

![image](https://github.com/cohstats/coh3-stats/assets/8086995/a864413c-652e-4549-87a9-581924141874)

I added these checks that if it's null it should not be displayed, so we can be sure what 

if("") is false, so it's going to work, but it's something we can think off how to handle across the app.